Makefile

```diff
@@ -99,7 +99,8 @@ endif
 # Install bash_completion
 install-etc:
 	@echo Installing bash completion
-	@sudo install -o root -g root -m 644 etc/bash_completion.d/juju2 /etc/bash_completion.d
+	@sudo install -o root -g root -m 644 etc/bash_completion.d/juju-2.0 /etc/bash_completion.d
+	@sudo install -o root -g root -m 644 etc/bash_completion.d/juju-version /etc/bash_completion.d
```

Comment:
These locations should probably be updated to the new `/usr/share/bash-completion/bash_completion` location. The split between /etc and there is also the cause of much confusion.